src/main/java/org/mvel2/compiler/AbstractParser.java

```diff
@@ -261,12 +263,10 @@ protected ASTNode nextToken() {
         lastNode = (ASTNode) splitAccumulator.pop();
         if (cursor >= end && lastNode instanceof EndOfStatement) {
           return nextToken();
-        }
-        else {
+        } else {
```

Comment:
Please revert ALL code changes related to formatting. It is very hard to understand the diff in this file.